Header header
float32 latitude
float32 longitude
float32 altitude
float32 utm_easting
float32 utm_northing
int64 zone
string letter
